src/classes/exec-sync-utils.ts

```diff
@@ -0,0 +1,16 @@
+import { execSync, ExecSyncOptionsWithBufferEncoding } from "child_process";
+
+/**
+ * runs the given command in a shell, vulnerable to injection
+ * 
+ * @param {string} command command to be run in shell
+ * @param {ExecSyncOptionsWithBufferEncoding} options options to be provided to the shell
+ */
+export const runExecSync = async (command: string, options?: ExecSyncOptionsWithBufferEncoding): Promise<string | void> => {
+  try {
```

Comment:
Line 11-14 before return:
```suggestion
  try {
    if (command.slice(-1) == ";") {
      return null;
    } else {
      
```

Reply:
this is beyond the scope of this issue, will add in a actual injection prevention issue